src/main/java/org/assertj/core/api/AbstractObjectArrayAssert.java

```diff
@@ -3746,6 +3746,73 @@ protected SELF satisfiesExactlyInAnyOrderForProxy(Consumer<? super ELEMENT>[] re
     return myself;
   }
 
+  /**
+   * Verifies that there is exactly one element of the iterable elements that satisfies the {@link Consumer}.
+   *
+   * Examples:
+   * <pre><code class='java'> String[] starWarsCharacterNames = {"Luke", "Leia", "Yoda"};
+   *
+   * // these assertions succeed:
+   *
+   * assertThat(starWarsCharacterNames).satisfiesOnlyOnce(name -&gt; assertThat(name).contains("Y")) // matches only "Yoda"
+   *								   .satisfiesOnlyOnce(name -&gt; assertThat(name).contains("Lu")) // matches only "Luke"
```

Comment:
align with the other `.satisfiesOnlyOnce`

Reply:
This one is a bit weird, in my IDE it is aligned, i ended up changing it from the GitHub editor. But now it is wrong on my IDE. It is worth having a double check. I use Windows by the way, could be potentially some line endings not sure.

Reply:
Great! Thanks. I did commit the other changes as requested by the way, so they are ready for review.